tutorials/algorithms/09_IQPE.ipynb

```diff
@@ -2,6 +2,7 @@
  "cells": [
```

Comment:
<p>Let's sample the circuit with Qiskit Aer's <code>Sampler</code> primitive, a local simulator without noise that run<strong>s</strong> locally.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1369/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>